zio-http-example/src/main/scala/example/datastar/SimpleHelloWorldExample.scala

```diff
@@ -0,0 +1,104 @@
+package example.datastar
+
+import zio._
+import zio.http._
+import zio.http.datastar._
+import zio.http.template2._
+
+object SimpleHelloWorldExample extends ZIOAppDefault {
+  val message = "Hello, world!"
+
+  val routes: Routes[Any, Response] = Routes(
+    // Main page route
+    Method.GET / Root          -> handler {
+      Response(
+        headers = Headers(
+          Header.ContentType(MediaType.text.html),
+        ),
+        body = Body.fromCharSequence(indexPage.render),
+      )
+    },
+    Method.GET / "hello-world" -> events {
+      handler {
+        ZIO.foreachDiscard(message.indices) { i =>
+          for {
+            _ <- ServerSentEventGenerator.executeScript(s"console.log('Sending character index: $i')")
+            _ <- ServerSentEventGenerator.patchElements(div(id("message"), message.substring(0, i + 1)))
+            _ <- ZIO.sleep(100.millis)
+          } yield ()
+        }
+      }
+    },
+  )
+
+  def indexPage = {
+    html(
+      head(
+        meta(charset("UTF-8")),
+        meta(name("viewport"), content("width=device-width, initial-scale=1.0")),
+        title("Datastar Hello World - ZIO HTTP Datastar"),
+        script(
+          `type` := "module",
+          src    := "https://cdn.jsdelivr.net/gh/starfederation/datastar@v1.0.0-RC.5/bundles/datastar.js",
+        ),
+        style.inlineCss(css),
+      ),
+      body(
+        dataOn.load := Js("@get('/hello-world')"),
```

Comment:
```suggestion
        dataOn.load := js"@get('/hello-world')",
```
We should never use the case class constructor in docs